aruna/api/storage/services/v2/info_service.proto

```diff
@@ -132,9 +143,14 @@ message GetPubkeysResponse {
 }
 
 message Announcement {
-  string id = 1;
-  string content = 2;
-  google.protobuf.Timestamp created_at = 3;
+  string announcement_id = 1;
+  storage.models.v2.AnnouncementType announcement_type = 2;
+  string title = 3;
+  string content = 4;
+  string created_by = 5;
+  google.protobuf.Timestamp created_at = 6;
+  string modified_by = 7;
+  google.protobuf.Timestamp modified_at = 8;
 }
 
 message GetAnnouncementsRequest {}
```

Comment:
Do we need another request that queries only a "short" section of an anouncement / blog entry ?
If they are really long it might be a lot of overhead to query the full content, but it could be necessary to query a list of anouncements (and type) by id or so.

Reply:
I was of the opinion that as a user you can independently use the first x characters/words/phrases of the content for such a purpose. But while we are at it, I can also add the `string teaser` field that e.g. summarizes the article in a short sentence.